app/controllers/bands_controller.rb

```diff
@@ -15,6 +15,33 @@ def create
     end
   end
 
+  def edit
+    @band = Band.find(params[:id])
+  end
+
+  def update
+    @band = Band.find(params[:id])
+
+    if @band.update(band_params)
+      flash[:notification] = "Changes have been made"
+      redirect_to root_path
```

Comment:
Would we possibly want to be redirected back to the show page for the band that was just updated?

Reply:
I would encourage you to tackle Kerrins comment!